### ISSUE TITLE
remove having to pick lock civ cars

### ISSUE THREAD

**Issue by EXO1111** [InZidiuZ/op-framework-issue-tracker#2582]
### Assurance

I have read the `README.md`

### Summary

my suggestion is that we remove having to pick lock a car 

### Reason

i think this would improve the server greatly. it gets pretty annoying being stranded and having to run from car to car just to get places. i understand how this can provide that realistic roleplay element to the server but i think i speak for everyone when i say its just frustrating. instead maybe replace it with a new pick locking system, one that allows us to enter the car but have to hit a certain key at a specific time; ive seen that in other cities

**Comment by CloudTheWolf** [comment]
> Disagree. There are already tons of options besides running from car to car...
> 
> * Walk
> * Wave a car down
> * Call a friend
> * Call a cab
> * Borrow a car
> * Buy a car
> * Find other ways to acquire local vehicles
> * Talk to people for even better ways to do crime things

Agreed, Getting access to a player owned vehicle, that most likely has been at least partially upgraded, should not be as easy as robbing a local vehicle (Either by Stand up, shooting the local and stealing the keys, or hotwiring a local vehicle)

**Comment by coalaura** [comment]
TL;DR: No
